app/controllers/admin/items_controller.rb

```diff
@@ -27,7 +30,10 @@ def edit
   end
 
   def update
+    category_collection = params[:item][:categories].reject {|x| x.empty?}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:empty?` as an argument to `reject` instead of a block.